### ISSUE TITLE
LFN (Long file Name) issue C:\Progra~1 == "C:\Program Files" with CMD command interpreter

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/windows-package-manager#100]
```
What steps will reproduce the problem?
1. Start, Run, cmd /k cd /d "%temp%\npackd-1.16.2" <OK>
2. npackdcl add -p=com.adobe.FlashPlayerActiveX
or
3. npackdcl add -p=com.adobe.FlashPlayer

What is the expected output? What do you see instead?
Expected:
package installed without error
What we get:
Error installing Flash Player ActiveX (Internet Explorer) 11.0.1.152: Process 
C:\Windows\system32\cmd.exe exited with the code 1
'C:\Program' is not recognized as an internal or external command
Error installing Flash Player (Firefox, Safari, Opera) 11.0.1.152: Process 
C:\Windows\system32\cmd.exe exited with the code 1
'C:\Program' is not recognized as an internal or external command


What version of the product are you using? On what operating system?
1.16.2
Windows XP Professional SP3 32bit ENU fresh install, no third party 
applications installed and no previous version of npackd

Please provide any additional information below.
Start, Run, cmd /k cd /d "%temp%" <OK> 
pushd C:\Program Files\Flash_Player_ActiveX_(Internet_Explorer) 
.WPM\install.bat

package is installed as expected!


Two more things:
package name for Flash Players should be:

p=com.adobe.FlashPlayerPlugin
Title: Flash Player Plugin

p=com.adobe.FlashPlayerActiveX
Title: Flash Player ActiveX

Any advanced user who use your program should know the difference between 
browsers: ActiveX is related to Microsoft closed source and Plugin to open 
source browsers.

I found it ugly to use parenthesis () for folder naming.
Altough Microsoft itself use it for differentiating Program Files (x86).

add more complexity in batch scripting.

```

Original issue reported on code.google.com by `bilou.ga...@gmail.com` on 19 Nov 2011 at 2:40

**Comment by GoogleCodeExporter** [comment]
```
have just renamed the packages. This really fixes it. Need to investigate this 
further.
There will be 1.16.3

BTW, thanks for testing the beta
```

Original comment by `tim.lebe...@gmail.com` on 19 Nov 2011 at 6:25
* Changed state: **Accepted**
* Added labels: **Milestone-End_Of_Month**

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.16.3
```

Original comment by `tim.lebe...@gmail.com` on 27 Nov 2011 at 10:04
* Changed state: **Fixed**
